src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -42,19 +42,19 @@ components:
       description: The SQS message for sending an application to print.
       type: object
       properties:
-        voterApplicationId:
+        sourceReference:
           type: string
-          format: uuid
-          description: The Voter Authority Certificate identifier
-          example: 14ab70e8-bd3b-400f-bd95-246caf9e4810
+          description: Reference in the source application that this print request relates to
         applicationReference:
           type: string
           description: The application reference as known by the citizen. Not guaranteed to be unique.
+        sourceType:
```

Comment:
Consistent with RCA and notifications-api, I have added a property `sourceType` that tells us what type of application this is for (ie. what the source "application" was - though @matt-wills-valtech hates the term source *application* 😁 )
It's an enum (defined below), currently with one value in it (`voter-card`), so we'll be able to use this, along with the `sourceReference` to communicate back to the source *application* 😁 if we need to (or indeed filter by source type when we do the print queue etc)

Reply:
Happy for you to name this how you see consistent.

Reply:
👍